src/disclosureMenu.js

```diff
@@ -204,23 +204,18 @@ class DisclosureMenu extends BaseMenu {
   /**
    * Handles click events throughout the menu for proper use.
    *
-   * Depending on what is supported either `touchstart` and `touchend` or
-   * `mousedown` and `mouseup` will be used for all "click" event handling.
+   * `pointerup` will be used for all "click" event handling.
    *
    * - Adds all event listeners listed in
    *   {@link BaseMenu#_handleClick|BaseMenu's _handleClick method}, and
-   * - adds a `touchend`/`mouseup` listener to the `document` so if the user
+   * - adds a `pointerup` listener to the `document` so if the user
    *   clicks outside of the menu it will close if it is open.
    *
    * @protected
    */
   _handleClick() {
     super._handleClick();
-
-    // Use touch over mouse events when supported.
-    const endEventType = isEventSupported("touchend", this.dom.menu)
-      ? "touchend"
-      : "mouseup";
+    const endEventType = "pointerup";
```

Comment:
Let's just get rid of this `const` since it's only used in one place and there is no longer any logic to figure which event to use.
```suggestion
```